docs/release/releases/release-1.6.md

```diff
@@ -32,4 +32,5 @@ After the `.0` release monthly patch release will be created.
 |-------------------------------------------|--------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Release Lead                              | TBD | TBD   |
 | Communications/Docs/Release Notes Manager | TBD | TBD   |
-| CI Signal/Bug Triage/Automation Manager   | TBD | TBD   |
\ No newline at end of file
+| CI Signal/Bug Triage/Automation Manager   | TBD | TBD   |
+| Release Writer                            | TBD | TBD   |
```

Comment:
sorry for the late review! this should also be renamed to `Maintainer` 

otherwise LGTM

Reply:
Thanks for catching that - opened a PR at https://github.com/kubernetes-sigs/cluster-api/pull/9013